bin/node/runtime/pangolin/src/weights/darwinia_staking.rs

```diff
@@ -102,6 +102,12 @@ impl<T: frame_system::Trait> darwinia_staking::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().writes(3 as Weight))
 			.saturating_add(T::DbWeight::get().writes((3 as Weight).saturating_mul(n as Weight)))
 	}
+	fn rebond(l: u32) -> Weight {
+		(46_569_000 as Weight)
```

Comment:
How is the weight calculated here.

Reply:
//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0